receiver/postgresqlreceiver/metadata.yaml

```
@@ -37,6 +57,52 @@ attributes:
     enum: [dead, live]
 
 metrics:
+  postgresql.bgwriter.buffers.allocated:
+    attributes: []
+    description: Number of buffers allocated.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{buffers}"
+  postgresql.bgwriter.buffers.writes:
+    attributes:
+      - bg_buffer_source
+    description: Number of buffers written
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{buffers}"
+  postgresql.bgwriter.checkpoint.count:
+    attributes:
+      - bg_checkpoint_type
+    description: The number of checkpoints performed
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: "{checkpoints}"
+  postgresql.bgwriter.duration:
+    attributes:
+      - bg_duration_type
+    description: Total time spent writing and syncing files to disk by checkpoints.
+    enabled: true
+    sum:
+      aggregation: cumulative
+      monotonic: true
+      value_type: int
+    unit: ms
+  postgresql.bgwriter.maxwritten.count:
```

```suggestion
  postgresql.bgwriter.maxwritten.count:
```

I think this name needs tweaking. The namespace here is `postgresql.bgwriter.maxwritten`. Typically the last word of a namespace is a noun, and the metric describes the noun in some way. Maybe just `postgresql.bgwriter.maxwritten` is appropriate here?